kombu/transport/SQS.py

```diff
@@ -583,24 +586,14 @@ def _on_messages_ready(self, queue, qname, messages):
                 msg_parsed = self._message_to_python(msg, qname, queue)
                 callbacks[qname](msg_parsed)
 
-    def _get_from_sqs(self, queue,
-                      count=1, connection=None, callback=None):
+    def _get_from_sqs(self, queue_name, queue_url,
+                      connection, count=1, callback=None):
         """Retrieve and handle messages from SQS.
 
         Uses long polling and returns :class:`~vine.promises.promise`.
         """
-        connection = connection if connection is not None else queue.connection
```

Comment:
and removal of this too

Reply:
This is also covered by the unit test in the latest commit